python/nav/bitvector.py

```diff
@@ -46,7 +49,7 @@ def __setitem__(self, posn, val):
         significant bit is regarded as bit 0 in this context.
         """
         val = val and 1 or 0
-        block = posn / 8
+        block = posn // 8
```

Comment:
Actually, upon testing, this breaks BitVector. Just a few lines down, `BitVector.__repr__()` uses `str()` on itself as part of the output, which appears to turn into an infinite recursion (under Python 2)..

Reply:
Uhmr, this comment doesn't belong on this line. Stupid GitHub...